app/poros/item.rb

```diff
@@ -0,0 +1,13 @@
+class Item
+  attr_reader :id,
+              :name,
+              :description,
+              :unit_price
+
+  def initialize(item_data)
+    @id          = item_data[:id]
+    @name        = item_data[:attributes][:name]
+    @description = item_data[:attributes][:description]
+    @unit_price  = item_data[:attributes][:unit_price]
+  end
+end
```

Comment:
THis goes for the merchant class too, but be careful naming an attribute "id", since poros automatically have a unique id assocoiated with them. THis could cause some issues when you're trying to access the id that you manually set later on.